lib/mixins/annotations.js

```diff
@@ -39,6 +39,19 @@ export default {
     return this.annotate(x, y, w, h, options);
   },
 
+  goTo(x, y, w, h, name, options) {
```

Comment:
It's fine use es syntax for default value: options = {}

Reply:
Done. And created PR for fix in all other occurrences https://github.com/foliojs/pdfkit/pull/948